### DISCUSSION TITLE
Want a way to make GFM text bigger

### DISCUSSION THREAD

**Discussion by Artoria2e5** [community/community#58531] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There are times you really want to drive a point across but neither _italics_ nor **bold** (or _**both**_) is enough to stop people from ignoring it and asking questions that have already been answered. In those cases you want a bigger font, but the only option on the website right now (given tag filtering / sanitization) is the heading levels, which is not very great for text flow.

An example can be seen at https://github.com/TheUsefulLists/UsefulMods/blob/fc84ef3/README.md, where the author @NordicGamerFE clearly intends to make sure the thing to be seen. But now we get a whole line across the screen, because it's an `<h2>`! That breaks the flow way more than any big centered text would!

It would be sweet if some type of `<big>` can be allowed through the filter. Yes, it's [very deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/big) because it serves absolutely zero semantic purpose as far as HTML is concerned, but it makes sense in this context because we have zero control over styling. (GH also allows `align=center`, even though it's deprecated, for the same reason.) If spec compliance is a problem, there can be a pass after the sanitizer to convert it into a more modern, CSS-based form.

Or there could be a new syntax for that. That sounds like it would take much more care to make sure it does not break anything else.

**Comment by NordicGamerFE** [comment]
<3 Thanks for the mention, One alternative atm tho very janky is to use images. I personally don't use this but it's possible to achieve the text size, font, or look you want.

**Reply by Artoria2e5** [reply]
I wanted to send you guys a PR but then realized there's literally no good way to do it. Oops.

Not a big fan of text-as-picture either: i know alt texts are a thing but it just feels wrong in general.
